sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -81,27 +63,25 @@ class KernelRidge(BaseEstimator, RegressorMixin):
       "Machine Learning: A Probabilistic Perspective", The MIT Press
       chapter 14.4.3, pp. 492-493
 
+    * An, Senjian, Wanquan Liu, and Svetha Venkatesh.
```

Comment:
This should only be in the KernelRidgeClassifier docstring.

But if you drop the BaseKernelRidge docstring, that would help you tease things apart

Reply:
Done, that paper now only appear in KernelRidgeclassifier,

About docstring, as I said: if you are really concerned about this, I can remove it, but I think there is one test that would complain.